### ISSUE TITLE
Request to adjust difficulty better

### ISSUE THREAD

**Issue by dimasbjg** [alfianrehanusa/balloon_popper_android#1]
Instead of using duration for difficulty. What about make the balloon move horizontally.

**Comment by rzfahm** [comment]
Yes, this is extremely serious issue. How can people live without balloon floating horizontally?

**Comment by alfianrehanusa** [comment]
Its a good idea. PR is wide open.
